EstateManagement/Factories/IModelFactory.cs

```diff
@@ -23,8 +23,16 @@ public interface IModelFactory
         /// Converts from.
         /// </summary>
         /// <param name="merchant">The merchant.</param>
+        /// <param name="merchantBalance">The merchant balance.</param>
         /// <returns></returns>
-        MerchantResponse ConvertFrom(Merchant merchant);
+        MerchantResponse ConvertFrom(Merchant merchant, MerchantBalance merchantBalance = null);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/stuart_ferguson1/EstateManagement/pullRequest?prid=5660966)